rocky-backend/DBtest.py

```
@@ -0,0 +1,95 @@
+# create valid user
+res = requests.post(f"{BASE}/users", json={
+    "name": "Jane",
+    "email": "jane@test.com",
+    "flash_id": "321",
+    "role": "student"
+})
+test("Create valid user", res.status_code == 200)
+
+
+#create user with missing fields
+res = requests.post(f"{BASE}/users", json={"name": "Bad"})
```

Ruff (0.15.9)

[error] Probable use of `requests` call without timeout

(S113)